docs/REST API/rest-api.md

```diff
@@ -4276,6 +4276,8 @@ OS vulnerability data is currently available for Windows and macOS. For other pl
 
 ### Get host's software
 
+> The [new keys/values](https://github.com/fleetdm/fleet/pull/19245/files) added in the app management features are in beta and may change. You can find the upcoming breaking changes [here](https://github.com/fleetdm/fleet/pull/19291).
```

Comment:
I think we should be more explicit than linking to changes. What is there now, and what is it changing to?

Reply:
Or we should at least link to the specific lines we are referencing. Otherwise very hard to understand what is being references at those links.

Reply:
@lukeheath great points. My latest commit pulls out links to the old (merged) PR and updates links to point to specific lines we are referencing.